changelog/1813.breaking.rst

```diff
@@ -0,0 +1 @@
+Changed the gyroradius function so that it takes relativistic effects into account.
```

Comment:
yeah I called it breaking just in case somehow someone wanted to keep the exact same behavior before and after

Reply:
Yeah, I absolutely see why! It's a tough question.

Reply:
I think it'd be fine for the changelog to be either a feature or a breaking change, so my inclination would be to keep it as a breaking change...since that'd be less work!